### DISCUSSION TITLE
Passat B8 2017 Alltrack 220tsi dataset problem

### DISCUSSION THREAD

**Discussion by Mex77** [jilleb/MQB-FPA#85] [Q&A]
Im on Conti 3Q0907530AC gateway now.. AA flashfile is impossible to find..

All datasets that I find are missing FPA for eco and offroad, so there is just to much to add..
Does anybody have dataset with:
Eco, comfort, normal, sport, individual and offroad.

**Comment by jilleb** [comment]
Check out the example datasets on mqbtools.nl/fpa

**Comment by Mex77** [comment]
I have, but cannot save/download them..
The files I make with script does not work on mqbtools.nl/fpa..
"Incorrect file size"

**Reply by jilleb** [reply]
If you open an example dataset and save it as raw binary, you can open it in the script.

And raw binary files can be uploaded to the site.

**Comment by Mex77** [comment]
500 Internal Server Error

**Reply by jilleb** [reply]
For which dataset? I just checked, nothing wrong when saving a dataset.

**Reply by Mex77** [reply]
Passat R 2020 DCC slider.
Will try another browser (on Chrome now)..

**Reply by Mex77** [reply]
Does not help with Passat 2020 DCC slider (NOT/don´t need Passat R 2020 DCC slider).

Most of them can be downloaded and passat and magotan can´t, no matter what (browser, cleared cookies..)

**Reply by jilleb** [reply]
Fixed it! Thanks for the report.

**Reply by Mex77** [reply]
Downloaded.. 

Thank you :)
